File: c/002-discount.c
#include <stdio.h>

int main() {
    float merchandise_value;
    float discount_percentage;

    printf("Insira o valor da mercadoria: ");
    scanf("%f", &merchandise_value);
    printf("Insira o percentual de desconto: ");
    scanf("%f", &discount_percentage);

    float discount_value = (merchandise_value * discount_percentage) / 100;
    float price = merchandise_value - discount_value;

    printf("Valor do desconto: R$ %.2f\n", discount_value);
    printf("Valor a ser pago: R$ %.2f\n", price);

    return 0;
}

File: python/002-discount.py
merchandise_value = float(input("Insira o valor da mercadoria: "))
discount_percentage = float(input("Insira o percentual de desconto: "))

discountValue = (merchandise_value * discount_percentage) / 100
price = merchandise_value - discountValue

print("Valor do desconto: R$ {:.2f}".format(discountValue))
print("Valor a ser pago: R$ {:.2f}".format(price))
